data/forecast.go

```diff
@@ -0,0 +1,60 @@
+package data
+
+import (
+	"time"
+
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	MinimumTimeScale = 0
+	MaximumTimeScale = 3600000
```

Comment:
Okay if we plan to (re?)introduce `timeScale`. Otherwise remove.

Reply:
In spreadsheet - there was no mention of a timeScale parameter.  We can put one in - but I do not think it will be filled out.  For now - I am leaving out.